tests/test_errorcallback.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the error callback"""
+
+import unittest
+import unittest.mock
+
+import rtmidi
+
+class RtMidiTestCase(unittest.TestCase):
```

Comment:
> What are the functions then?

Tests?

You don't need to use `TestCase` in the name. I just used it in `test_rtmidi.py`, because I'm sub-classing `unittest.TestCase`. But I'm leaning towards using pytest these days, anyways, so sub-classing `unittest.TestCase` wouldn't be strictly necessary.